Test/PerformanceCollector/FunctionalTests/app.config

```diff
@@ -1,14 +1,22 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <system.serviceModel>
       <bindings />
       <behaviors>
         <endpointBehaviors>
           <behavior name="httpBinding">
-            <webHttp/>
+            <webHttp />
           </behavior>
         </endpointBehaviors>
       </behaviors>
       <client />
     </system.serviceModel>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-6.0.0.0" newVersion="6.0.0.0" />
```

Comment:
I know it's not related to this changelist but I noticed this was out of date and we should probably update it to version 10 as well at some point.